src/lib/arch/aarch64/Instruction_decode.cc

```diff
@@ -341,7 +337,6 @@ void Instruction::decode() {
         exceptionEncountered_ = true;
         exception_ = InstructionException::UnmappedSysReg;
         // Clear any registered operands
-        sourceRegisterCount = 0;
         destinationRegisterCount = 0;
```

Comment:
Wondering if we should just return at this stage rather than setting reg counts to zero

Reply:
changed